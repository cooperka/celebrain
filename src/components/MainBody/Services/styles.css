.Services {
  margin-bottom: 4em;
  display: flex;
  flex-direction: column;
  justify-content: center;
  font-weight: bold;
}

.Services .title {
  margin: 1rem;
  font-size: 4em;
}

.Services .subtitle {
  margin: 1rem auto;
  font-size: 2em;
  max-width: 20em;
}

.Services .grid-container {
  display: grid;
  grid-template-columns: 1fr 1fr;
  margin: auto;
}

.Services .grid-item {
  padding: 1em 2em;
  font-size: 1.2em;
}
