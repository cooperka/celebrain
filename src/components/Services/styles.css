.Services .grid-container {
  display: grid;
  grid-template-columns: 1fr;
  margin: 0 auto 1rem auto;
}

.Services .grid-item {
  padding: 1.2rem 2rem;
  font-size: 1.6rem;
}
