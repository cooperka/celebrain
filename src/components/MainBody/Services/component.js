// @flow

import React from 'react';

import './styles.css';

function Services() {
  return (
    <div className="Services">
      <div className="title">Services</div>
    </div>
  );
}

export default Services;
