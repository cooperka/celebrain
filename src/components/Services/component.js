// @flow

import React from 'react';

import './styles.css';

function Services() {
  return (
    <div className="Services">
      <div className="title">Services</div>
      <div className="subtitle">Software development</div>
      <div className="grid-container">
        <div className="grid-item">Web apps</div>
        <div className="grid-item">Mobile apps</div>
        <div className="grid-item">User testing</div>
        <div className="grid-item">Project planning</div>
      </div>
      <div className="subtitle">Small business development</div>
      <div className="grid-container">
        <div className="grid-item">Strategic planning</div>
        <div className="grid-item">Team building</div>
      </div>
    </div>
  );
}

export default Services;
