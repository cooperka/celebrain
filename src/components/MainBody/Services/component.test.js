// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Services from './component';

it('renders', () => {
  const tree = renderer.create(<Services />);
  expect(tree.toJSON()).toMatchSnapshot();
});
