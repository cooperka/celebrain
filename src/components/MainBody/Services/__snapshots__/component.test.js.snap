// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Services"
>
  <div
    className="title"
  >
    Services
  </div>
  <div
    className="subtitle"
  >
    End-to-end software development
  </div>
  <div
    className="grid-container"
  >
    <div
      className="grid-item"
    >
      Web apps
    </div>
    <div
      className="grid-item"
    >
      Mobile apps
    </div>
    <div
      className="grid-item"
    >
      User testing
    </div>
    <div
      className="grid-item"
    >
      Strategic planning
    </div>
  </div>
</div>
`;
